dronekit/__init__.py

```diff
@@ -110,6 +111,7 @@ def __init__(self, master, status_printer=None):
         self.epv = None
         self.satellites_visible = None
         self.fix_type = None  # FIXME support multiple GPSs per vehicle - possibly by using componentId
+        self.ekf_ok = False
```

Comment:
what do you think of making an object or class for `ekf` and exposing `status` attribute, we might want to add more data to this object later